message_sender/models.py

```diff
@@ -39,8 +39,10 @@ class Outbound(models.Model):
     Delivered is set to true when ack received because delivery reports patchy
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    to_addr = models.CharField(null=False, blank=False, max_length=500,
+    to_addr = models.CharField(null=True, blank=False, max_length=500,
```

Comment:
I think we want this to be `null=False, blank=True` for all the fields, since it's the Django convention.

The reason for this is that when you're trying to create a message object on a form, say in the django admin, char fields render as text input boxes. For a text input box, you can leave it blank, but you can't make it null. So that's why in django, we prefer to use blank as our "no value" choice, rather than null.